hw/dv/tools/dvsim/testplans/sec_cm_testplan.hjson

```diff
@@ -18,7 +18,7 @@
             - Check that err_code/fault_status is updated correctly and preserved until reset.
             - Check the following operation should be failed if applicable.'''
       milestone: V2
-      tests: ["{name}_harden_countermeasures"]
+      tests: ["{name}_sec_cm"]
     }
     {
       name: redundant_coding_fsm_check
```

Comment:
Let's say this testplan is imported for some block containing the prim_counter but not the prim_fsm. In the published regression results page, it will show ALL of these testpoints (and they will all show as passing, since the same test is associated with all these testpoints), even though it only has the prim_counter. That feels incorrect to me. That is why I was suggesting splitting each of these into its own testplan.